### ISSUE TITLE
Series name isn't updated properly (wrong casing and missing apostrophe)

### ISSUE THREAD

**Issue by kaelspencer** [dbr/tvnamer#202]
I upgrade from what I think was tvnamer 3.0.0 with tvdb_api 3.0.2. I no longer have the old virtual environments, so I can't be sure. Upon upgrading to tvnamer 3.0.2 with tvdb_api 3.1.0, I'm seeing incorrect file names generated, specifically with the series name.

Given the file `bobs.burgers.s11e21.mkv` I expect and previously got `Bob's Burgers - [11x21] - Tell Me Dumb Thing Good.mkv`. I'm now getting `bobs burgers - [11x21] - Tell Me Dumb Thing Good.mkv`. The series name is not cased correctly and is missing the apostrophe.

I spun up a new Ubuntu 21.04 vm for these tests so there is no hidden config file anywhere.

## Broken with tvnamer 3.0.3 and tvdb_api 3.10

```
$ python --version
Python 3.9.4

$ tvnamer --version
tvnamer version: 3.0.3
tvdb_api version: 3.1.0

$ touch bobs.burgers.s11e21.mkv

$ tvnamer --dry-run bobs.burgers.s11e21.mkv
####################
# Starting tvnamer
# Found 1 episode
####################
# Processing file: bobs.burgers.s11e21.mkv
# Detected series: bobs burgers (season: 11, episode: 21)
TVDB Search Results:
1 -> Bob's Burgers [en] # http://thetvdb.com/?tab=series&id=194031 (default)
Automatically selecting only result
####################
Old filename: bobs.burgers.s11e21.mkv
New filename: bobs burgers - [11x21] - Tell Me Dumb Thing Good.mkv
bobs.burgers.s11e21.mkv will be renamed to bobs burgers - [11x21] - Tell Me Dumb Thing Good.mkv

####################
# Done
```

## Working with tvnamer 3.0.2 and tvdb_api 3.0.2

```
$ python --version
Python 3.9.4

$ tvnamer --version
tvnamer version: 3.0.2
tvdb_api version: 3.0.2

$ touch bobs.burgers.s11e21.mkv

$ tvnamer --dry-run bobs.burgers.s11e21.mkv 
####################
# Starting tvnamer
# Found 1 episode
####################
# Processing file: bobs.burgers.s11e21.mkv
# Detected series: bobs burgers (season: 11, episode: 21)
TVDB Search Results:
1 -> Bob's Burgers [en] # http://thetvdb.com/?tab=series&id=194031 (default)
Automatically selecting only result
####################
Old filename: bobs.burgers.s11e21.mkv
New filename: Bob's Burgers - [11x21] - Tell Me Dumb Thing Good.mkv
bobs.burgers.s11e21.mkv will be renamed to Bob's Burgers - [11x21] - Tell Me Dumb Thing Good.mkv

####################
# Done
```

**Comment by kaelspencer** [comment]
Typo here?
https://github.com/dbr/tvnamer/blob/e673514b6f72f501904848d9a064621d7bb74abf/tvnamer/utils.py#L655

**Comment by dbr** [comment]
Thanks again for the PR. Fixed in tvnamer 3.0.4 🥳
